### DISCUSSION TITLE
I get: Unsupported URL Type "workspace:": workspace:*  when trying to install @blueprintjs/core

### DISCUSSION THREAD

**Discussion by fabyyiola** [vercel/turborepo#5687] [Help]
Hi, im new to turbo, im trying to start a new project with it that will use blueprint js components library, i used your example with-tailwind but when i try to install any package on the root of my monorepo i get the same message:

Unsupported URL Type "workspace:": workspace:* 

I know it must be something super silly probably a config or something but well im new to this library and cant find anything online =(( i would appreciate any help, thank you!

**Comment by anthonyshew** [comment]
Could you provide a reproduction for this? Would love to help out. 🙏

**Comment by anthonyshew** [comment]
Going to close due to staleness.
